godot-ffi/src/lib.rs

```diff
@@ -290,8 +290,16 @@ pub unsafe fn deinitialize() {
 fn print_preamble(version: GDExtensionGodotVersion) {
     let api_version: &'static str = GdextBuild::godot_static_version_string();
     let runtime_version = read_version_string(&version);
-
-    println!("Initialize godot-rust (API {api_version}, runtime {runtime_version})");
+    let checks_mode = if cfg!(checks_at_least = "paranoid") {
+        "paranoid"
+    } else if cfg!(checks_at_least = "balanced") {
+        "balanced"
+    } else if cfg!(checks_at_least = "fast-unsafe") {
+        "fast-unsafe"
+    } else {
+        unreachable!();
+    };
+    println!("Initialize godot-rust (API {api_version}, runtime {runtime_version}, safety checks {checks_mode})");
```

Comment:
Great idea! :+1: 

Could you extract this into a function? Might come in handy somewhere else...
```rs
fn safety_checks_string() -> &'static str { ... }
```